src/linear.js

```diff
@@ -4,53 +4,51 @@ import {initRange} from "./init.js";
 import tickFormat from "./tickFormat.js";
 
 export function linearish(scale) {
-  var domain = scale.domain;
+  const domain = scale.domain;
```

Comment:
While `const` is cool and all, these changes make `src/linear.js` inconsistent with the rest of the modules in the packages, which all use strictly `var`.

Reply:
I'm happy to revert these changes. I saw that https://github.com/d3/d3-scale/blob/master/src/ordinal.js already uses const so I figured it would be okay to move towards es6 here as well.